recon.sh

```diff
@@ -2,9 +2,11 @@
 
 # default settings
 method=""
-sim_file="out/sim.root"
-rec_file="out/rec.root"
-aux_file="geo/irt-drich.root"
+#sim_file="out/irt.edm4hep.root"
+sim_file=$2
+#rec_file="out/rec.root"
+rec_file=$3
+aux_file="./irt-drich.root"
```

Comment:
These defaults are meant to be compatible with default filenames from `simulate.py` and `scripts/create_irt_auxfile.sh`, I think we should leave them as is. The `getopts` below allows the user to specify these files, and will allow for more flexibility of this script.